### ISSUE TITLE
I use this code (wifi),but it can't show wifi icon 

### ISSUE THREAD

**Issue by anfan4730** [romkatv/powerlevel10k#2621]
Now what can i operation ,let it can display correct icon for me and after realize this function I still also want view network speed for real-time. of crouse ,real-time network speed show is this function comes with it

**Comment by anfan4730** [comment]
My job type is network part ,bring this question to everybody I feel  very sorry may be my skill so bad cause i'm not able to independent solve this problem

**Comment by romkatv** [comment]
I don't understand your question. Can you rephrase it? If English isn't your native language, consider writing in your native language and using https://deepl.com/ to translate to English.

**Comment by anfan4730** [comment]
> I don't understand your question. Can you rephrase it? If English isn't your native language, consider writing in your native language and using https://deepl.com/ to translate to English.

I am using https://deepl.com/to translate to English.
This is re-translated
I am using this code in the "typeset-g POWERLEVEL9K_LEFT_PROMPT_ELEMENTS=()" module (wifi) and it is not displaying the wifi icon, but now I have connected to my wireless network!
Now what can I do to manipulate it so that it can display the correct icon for me, after implementing this I also want to see the network speed in real time. Of course, the real-time network speed display comes with this feature!
For this code "wifi       #wifi speed"

**Comment by romkatv** [comment]
This is a dup of https://github.com/romkatv/powerlevel10k/issues/2595.
